internal/mode/static/nginx/config/upstreams_template.go

```diff
@@ -7,6 +7,33 @@ package config
 // https://github.com/nginxinc/nginx-gateway-fabric/issues/483
 const upstreamsTemplateText = `
 {{ range $u := . }}
+upstream {{ $u.Name }} {
+    # if the keepalive directive us present, it is necessary to activate the load balancing method before the directive
```

Comment:
This will end up being rendered in the nginx conf, so maybe we put this comment outside of the template. Also `us` should be `is`.

Reply:
Oh ya, we probably don't want that rendered. My bad.